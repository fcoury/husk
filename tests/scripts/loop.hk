for i in 1..3 {
  println(i)
}

for i in 1..=5 {
  println(i)
}

for i in 1..10 {
  if i == 3 {
    break;
  }
  println(i)
}

for i in 1..=10 {
  if i == 3 {
    continue;
  }
  println(i)
}
