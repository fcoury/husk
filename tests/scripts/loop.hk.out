1
2
1
2
3
4
5
1
2
1
2
4
5
6
7
8
9
10
0
3
6
9
12
15
18
